egs/iwslt22_ta/ST/pruned_transducer_stateless5/decode.py

```
@@ -0,0 +1,948 @@
+    test_set_wers = dict()
+    
+    for key, results in results_dict.items():
+        recog_path = (
+            params.res_dir / f"{test_set_name}-{key}-{params.suffix}.txt"
+        )
+        results = sorted(results)
+        store_translations(filename=recog_path, texts=results)
+        logging.info(f"The transcripts are stored in {recog_path}")
+
+
+def save_results(
```

Ruff (0.12.2)

Redefinition of unused `save_results` from line 674

(F811)